leaf-ui/plugins/slider/nouislider.min.css

```diff
@@ -1,4 +1,316 @@
\ No newline at end of file
+}
+
+.noUi-target{
+    position:relative;
+    direction:ltr
+
+}
+
+.noUi-base{
+    width: 100%;
+    height: 100%;
+    position: relative;
+    z-index:1;
+
+}
+
+.noUi-origin{
+    position:absolute;
+    right:0;
+    top:0;
+    left:0;
+    bottom:0
+
+}
+
+.noUi-handle{
+    position:relative;
+    z-index:1;
+
+}
+
+.noUi-stacking .noUi-handle{
+    z-index:10
+
+}
+
+.noUi-state-tap .noUi-origin{
+    -webkit-transition:left .3s,top .3s;transition:left .3s,top .3s
+
+}
+
+.noUi-state-drag *{
+    cursor:inherit!important
+
+}
+
+.noUi-base{
+    -webkit-transform:translate3d(0,0,0);
+    transform:translate3d(0,0,0)
+
+}
+
+.noUi-horizontal{
+    height:18px
+
+}
+
+.noUi-horizontal .noUi-handle{
+    width:34px;
+    height:28px;

```

Comment:
Is the previous height for `.noUi-horizontal ` in line 65 being overwritten? 

And is the top in line 73 being overwritten/overwriting the top in line 298

Reply:
I did not wrote the code, so I am not sure if I should touch it beyond making it readable for future use.